src/main/resources/data/spectrum/modonomicon/books/guidebook/entries/creating_life/creature_manipulation.json

```diff
@@ -84,11 +86,6 @@
       "type": "spectrum:spirit_instiller_crafting",
       "recipe_id": "spectrum:spirit_instiller/head_fusion/variant_changing/shulker_pink",
       "text": "book.spectrum.guidebook.creature_manipulation.shulkers.text"
-    },
-    {
-      "type": "spectrum:spirit_instiller_crafting",
-      "recipe_id": "spectrum:spirit_instiller/head_fusion/variant_changing/fox_arctic",
-      "text": "book.spectrum.guidebook.creature_manipulation.foxes.text"
```

Comment:
"book.spectrum.guidebook.creature_manipulation.foxes.text" is not used anymore and can be removed from the lang file

Reply:
ah whoops, forgot about that